src/sentry/workflow_engine/migrations/0045_migrate_issue_alerts.py

```diff
@@ -2167,60 +2171,76 @@ def _create_workflow_actions(if_dcg: Any, actions: list[dict[str, Any]]) -> None
         DataConditionGroupAction.objects.bulk_create(dcg_actions)
 
     # EXECUTION STARTS HERE
-    for project in RangeQuerySetWrapperWithProgressBarApprox(Project.objects.all()):
-        organization = project.organization
-        error_detector, _ = Detector.objects.get_or_create(
-            type="error",
-            project=project,
-            defaults={"config": {}, "name": "Error Detector"},
-        )
-
-        with transaction.atomic(router.db_for_write(Rule)):
-            rules = Rule.objects.select_for_update().filter(project=project)
+    backfill_key = "backfill_workflow_engine_issue_alerts"
+    redis_client = redis.redis_clusters.get(settings.SENTRY_MONITORS_REDIS_CLUSTER)
```

Comment:
@wedamija is this still a reasonable cluster to use?

Reply:
Yeah, probably we should stop using it at some point, but it's not like it's adding a lot of load to the cluster